### ISSUE TITLE
Export Proxyman Settings and Import On Another Mac

### ISSUE THREAD

**Issue by philosopherdog** [ProxymanApp/Proxyman#1550]
## Description
Export / Import Proxyman setup

Is this possible?

**Comment by NghiaTranUIT** [comment]
Yes, Proxyman has this feature to export your setting to different Mac:
- By GUI: https://docs.proxyman.io/basic-features/import-export-settings
- By Command Line: https://docs.proxyman.io/command-line#1.-export-proxyman-config
